### DISCUSSION TITLE
Website Missing in Webstudio Dashboard / URL is live

### DISCUSSION THREAD

**Discussion by maximosandoval** [webstudio-is/webstudio-community#155] [General]
- Webstudio dashboard is empty and contains no sites
- The URL is live and has been verified
- URL was previously working just fine
- URL is run thru Cloudflare per Webstudio's best recommendations
- I did not delete or remove the website from the dashboard - the URL is still functional and can be found

<img width="973" alt="webstudio-error" src="https://github.com/webstudio-is/webstudio-community/assets/80641742/228a4e55-c661-4e99-933b-fa7807fc3a82">

<img width="1311" alt="webstudio-error-1" src="https://github.com/webstudio-is/webstudio-community/assets/80641742/9528583f-ebee-4090-ba72-9cf434f99777">

<img width="1061" alt="webstudio-error-2" src="https://github.com/webstudio-is/webstudio-community/assets/80641742/1ff934a8-a86a-457c-a62c-7e2d99b27b3d">

<img width="929" alt="webstudio-error-3" src="https://github.com/webstudio-is/webstudio-community/assets/80641742/de2cff39-eb71-436d-91c5-4551619ecf79">

**Comment by kof** [comment]
Are you logging in with the wrong account? check your email in the dashboard profile menu.
